### ISSUE TITLE
Changing config.display_name_methods doesn't seem to work

### ISSUE THREAD

**Issue by Salet** [activeadmin/activeadmin#3805]
I encountered a name conflict trying to integrate ActiveAdmin 1.0.0pre and Spreecommerce 2.4.3 into one Rails 4 app. The main issue is that ActiveAdmin searches for display methods in following order:

    :display_name, :full_name, :name, :username, :login, :title, :email, :to_s

and Spree defines a global helper display_name here:
https://github.com/spree/spree/blob/master/backend/app/helpers/spree/admin/stock_locations_helper.rb

That leads to errors in said helper when accessing ActiveAdmin. I already described it in deatils on SO:
http://stackoverflow.com/questions/28583008/spreecommerce-and-activeadmin-display-name-conflict

The first thing that I tried is changing display_name_methods in ActiveAdmin initializer:

    config.display_name_methods = [:full_name, :name, :username, :login, :title, :email, :to_s]

But that doesn't seem to have any effect on which methods gets called. I restarded the server of course.

Any hints?

**Comment by timoschilling** [comment]
There is a irritation between two different methods:
1. ActiveAdmin has a helper `display_name`
2. a method that you can define on your Model (default is `display_name`)

The last one can be configured via `config.display_name_methods`.

But your problem is that Spree overwrites the `display_name` helper, not the Model method.

**Comment by timoschilling** [comment]
I will dive into both codes and take a look wich one is the evil one.

**Comment by Salet** [comment]
Thanks @timoschilling. Shall I x-post this to Spree's issue tracker?

**Comment by timoschilling** [comment]
From my point of view this is Spree's fault.

I have tried this:
```ruby
module ApplicationHelper
  def display_name *args
    raise
  end
end
```
And I doesn't overwrite ActiveAdmin's `display_name` helper.

**Comment by timoschilling** [comment]
closed while spree/spree#6224 has solved the problem.
